examples/custom-client-cxx/daemon.cpp

```diff
@@ -98,7 +98,7 @@ int cmd_daemon(std::string local_repo_path) {
         continue;  // There's no point trying to look for an update
       }
 
-      auto latest = res.GetLatest();
+      auto latest = res.SelectTarget();
```

Comment:
yep, I though about that too. Originally, I've removed the GetLatest method, but then re-added it to keep backward compatibility. Since we are going to keep it, there is no need to do change GetLatest -> SelectTarget globally.

Reply:
Done, I've reverted original `GetLatest` calls to the original method.